src/shared/utils/ast/nodes.ts

```diff
@@ -0,0 +1,25 @@
+import {get} from 'lodash'
```

Comment:
This file has only 2 new functions.
* isWindowPeriodVariableNode, isVariableDeclaration
* the rest is moved code